packages/world/src/Proxy.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v5.0.0) (proxy/Proxy.sol)
```

Comment:
we could link the full github permalink here

Reply:
I think for these imports it's cleaner to copy-paste as is. Plus we'd have to change other ones (ERC20, ECDSA etc.)

Reply:
I just meant putting the permalink to github in the comment on the top